### ISSUE TITLE
notify for all pokemon

### ISSUE THREAD

**Issue by iTayGNR** [AHAAAAAAA/PokemonGo-Map#2014]
How can I automatically add all pokemons to "Notify of Pokémon"?

**Comment by Darkrift411** [comment]
I agree. Filling out this list sucks. I would like checkboxes instead of the ul. Maybe even a "add all" (and get rid of the 151+ entries for now) and the ability to remove the few you do not want.

**Comment by philipp-r** [comment]
#1973

**Comment by tegob** [comment]
…or just notify for every Pokémon that is not in the ignore list.
